doc/guides/contributing/pull-requests.md

```diff
@@ -176,14 +176,10 @@ Sample complete commit message:
 ```txt
 subsystem: explain the commit in one line
 
-Body of commit message is a few lines of text, explaining things
-in more detail, possibly giving some background about the issue
-being fixed, etc.
-
-The body of the commit message can be several paragraphs, and
-please do proper word-wrap and keep columns shorter than about
-72 characters or so. That way, `git log` will show things
-nicely even when it is indented.
+The body of the commit message can one or more paragraphs, explaining things in
```

Comment:
```suggestion
The body of the commit message should be one or more paragraphs, explaining things in
```

Reply:
I saying `should`, it might indicate that committer has to write details
Some commit messages without details might be good enough on their own, should we replace this back to `can be`

Reply:
I think "should be" (as opposed to "must be") conveys that there really *should* be a commit message body but that it is not absolutely required. Using "can be" makes it sound up to the committer, and while that may be strictly true, I think we should encourage providing a message body. So I prefer "should" over "can".